lms/djangoapps/courseware/access.py

```diff
@@ -587,6 +540,9 @@ def can_load():
                 or _can_access_descriptor_with_start_date(user, descriptor, course_key)
             )
         )
+        if user.is_authenticated():
```

Comment:
Oh, it looks like this moved? I think the order _may_ be important;
it shouldn't try to get a response if this check already failed.